config/remote-config-dev.json

```diff
@@ -1,5 +1,28 @@
 {
-	"country_codes_to_region_codes": [{"countryCode":"AU","regionCodes":["my"]},{"countryCode":"CN","regionCodes":["my"]},{"countryCode":"HK","regionCodes":["my"]},{"countryCode":"IN","regionCodes":["my"]},{"countryCode":"ID","regionCodes":["my"]},{"countryCode":"JP","regionCodes":["my"]},{"countryCode":"PH","regionCodes":["my"]},{"countryCode":"KR","regionCodes":["my"]},{"countryCode":"TH","regionCodes":["my"]},{"countryCode":"VN","regionCodes":["my"]},{"countryCode":"SG","regionCodes":["sg"]},{"countryCode":"US","regionCodes":["us"]},{"countryCode":"MY","regionCodes":["my"]},{"countryCode":"NO","regionCodes":["no"]}],
-	"region_groups": [{"name":"Asia","description":"Southeast asia & pacific","backgroundColor":"lipstick","isPreview":false,"countryCodes":["SG","MY"]},{"name":"The Americas","description":"Latin & north america","backgroundColor":"azul","isPreview":true,"countryCodes":[]},{"name":"Loltel","description":"Only certain people should see this","backgroundColor":"yellow","isPreview":true,"countryCodes":["NO"]}]
-	"feature_flags": {"enableStripeInsteadOfApplePay":false}
+  "parameters": {
+    "country_code_to_region_codes": {
+      "defaultValue": {
+        "value": "[{\"countryCode\":\"AU\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"CN\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"HK\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"IN\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"ID\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"JP\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"PH\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"KR\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"TH\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"VN\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"SG\",\"regionCodes\":[\"sg\"]},{\"countryCode\":\"US\",\"regionCodes\":[\"us\"]},{\"countryCode\":\"MY\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"NO\",\"regionCodes\":[\"no\"]}]"
+      }
+    },
+    "region_groups": {
+      "defaultValue": {
+        "value": "[{\"name\":\"Asia\",\"description\":\"Southeast asia & pacific\",\"backgroundColor\":\"lipstick\",\"isPreview\":false,\"countryCodes\":[\"SG\",\"MY\"]},{\"name\":\"The Americas\",\"description\":\"Latin & north america\",\"backgroundColor\":\"azul\",\"isPreview\":true,\"countryCodes\":[]},{\"name\":\"Loltel\",\"description\":\"Only certain people should see this\",\"backgroundColor\":\"yellow\",\"isPreview\":true,\"countryCodes\":[\"NO\"]}]"
+      }
+    },
+    "feature_flags": {
+      "defaultValue": {
+        "value": "{\"enableStripeInsteadOfApplePay\":false}"
+      }
+    }
+  },
+  "version": {
+    "versionNumber": "37",
+    "updateTime": "2019-10-30T12:40:37.554Z",
+    "updateUser": {
+      "email": "firebase-adminsdk-robwm@pi-ostelco-dev.iam.gserviceaccount.com"
```

Comment:
Who is this user?   Some firebase admin user, or one of ours?

Reply:
My best guess would be that this is the author of the update, and the author is based on the service account from firebase.

Reply:
This exists in the service-account.json under `client_email`